bats/no-repo.bats

```diff
@@ -53,6 +53,34 @@ teardown() {
     [[ "$output" =~ "gc - Cleans up unreferenced data from the repository." ]] || false
 }
 
+@test "check all commands for valid help text" {
+    # pipe all commands to a file
+    # cut -s suppresses the line if it doesn't contain the delim
+    dolt | cut -f 1 -d " - " -s | sed "s/ //g" > all.txt
+
+    # filter out commands without "-h"
+    cat all.txt \
+        | sed "s/remote//g"    \
```

Comment:
Odd that these don't have help text. Probably should have a skipped test that doesn't omit these commands

Reply:
remote actually does have help text, I removed that from the filter. The rest are subcommand handlers, so if you give them `-h` or anything invalid they'll print their usage